### PR TITLE
Allow debug in CADDY_GLOBAL_OPTIONS

### PR DESCRIPTION
**Pull request by chu121su12**
This PR let `debug` option to be appended in CADDY_GLOBAL_OPTIONS. See https://caddyserver.com/docs/caddyfile/options#debug

### CODE REVIEW

File: src/Commands/StartFrankenPhpCommand.php [hunk 1]
```diff
@@ -95,7 +95,11 @@ public function handle(ServerProcessInspector $inspector, ServerStateFile $serve
             'LARAVEL_OCTANE' => 1,
             'MAX_REQUESTS' => $this->option('max-requests'),
             'REQUEST_MAX_EXECUTION_TIME' => $this->maxExecutionTime(),
-            'CADDY_GLOBAL_OPTIONS' => ($https && $this->option('http-redirect')) ? '' : 'auto_https disable_redirects',
+            'CADDY_GLOBAL_OPTIONS' => trim(sprintf(
+                '%s %s',
```
Comment (dunglas) [inline src/Commands/StartFrankenPhpCommand.php]:
```suggestion
                "%s\n%s",
```

**Review by dunglas — COMMENTED** [review]
Alternatively, to simplify the code, we could introduce more environment variables for these options (like `DEBUG` and `AUTO_HTTPS`).

**Comment by chu121su12** [comment]
AUTO_HTTPS is already part of `http-redirect`. Added `server-debug` option only for FrankenPHP for now

**Comment by taylorotwell** [comment]
I would actually like to see these become env vars otherwise we may end up with a lot of FrankenPHP specific CLI options?

**Comment by driesvints** [comment]
Going to close this one now as it seems stale. Feel free to resend this once you've made the necessary changes, thanks.

**Pull request closed without merging**